### ISSUE TITLE
CRAN release

### ISSUE THREAD

**Issue by nevrome** [nevrome/rdoxygen#8]
@mlysy 

Sorry for disappearing for 10+ weeks. I think we can release this new version 2.0.0 of rdoxygen to CRAN. I updated the `NEWS.md` and the `cran-comments.md`.

Before we do so: Do you want to become the main maintainer of this package? I realized that I did not work with Rcpp for a long time now and I also believe you understand this package better than I do. I would love to stay in touch but pass the main responsibility on to you.

**Comment by mlysy** [comment]
Hi @nevrome, nice to hear from you.

I agree that we should go ahead and release rdoxygen 2.0.0 on CRAN.  I'm happy to take over maintenance of the package, thanks for offering.

I'll admit that, while I still think that C++ documentation in R packages is a critical element error-free, portable, and efficient C++ code, I'm a bit less excited about doxygen vignettes.  The main reason is that doxygen HTML documentation is rather large, adding ~300Kb as an upfront cost (although the extra Kb per documentation line is very little).

All this to say that I've yet to include doxygen documentation in any of my recent CRAN releases (although the doxygen markup is still there :).  I'm working on a satisfactory solution to this, but I think the quickest approach would be to provide a simple mechanism to include the doxydoc in an automatically-generated [**pkgdown**](https://pkgdown.r-lib.org/) website.

Either way, I'm very glad to stay in touch :)

**Comment by nevrome** [comment]
OK - excellent. pkgdown might be a great extension for the future!

For the procedure: We somehow have to [change the maintainership on CRAN](https://stackoverflow.com/questions/39223320/transferring-maintainership-of-an-r-package-on-cran). I suggest we try to do this as follows:

1. I transfer this repo on github to you. 
2. You edit the DESCRIPTION accordingly and maybe add a sentence about the change in the cran-comments, the NEWS and the README. 
3. You submit. 
4. If I get contacted after the submission I will confirm the change.

**Comment by nevrome** [comment]
Oh - You have to get rid of your fork. I can't transfer the repo on github if mlysy/rdoxygen already exists.

**Comment by mlysy** [comment]
Thanks for looking into this.  mlysy/rdoxygen has been deleted.

**Comment by thk686** [comment]
> I'll admit that, while I still think that C++ documentation in R packages is a critical element error-free, portable, and efficient C++ code, I'm a bit less excited about doxygen vignettes. The main reason is that doxygen HTML documentation is rather large, adding ~300Kb as an upfront cost (although the extra Kb per documentation line is very little).

Is it not easy to simple exclude the generated output using .Rbuildignore?

**Comment by mlysy** [comment]
Unfortunately not without excluding the vignette altogether.  That is, when you obtain a package from CRAN the vignettes do not get built.  They are pre-built and shipped inside `inst/doc`.

**Comment by thk686** [comment]
Yes, that is what I meant -- exclude the vignette altogether. I think it
will still work with pkgdown on github but you would not include the
vignette in CRAN (although I have not tested that setup).
